Solutions/Azure Storage/Analytic Rules/Detection of data exfiltration activities from Azure storage.yaml

```diff
@@ -0,0 +1,51 @@
+id: 7e6we089-e756-40fb-r023-c8e8864e456z
+name: Detection of data exfiltration activities from Azure storage
+description: |
+  'Detection of data exfiltration activities from Azure storage accounts, based on anomaly detection mechanism.'
+severity: Medium
+status: Available
+requiredDataConnectors:
+  - connectorId: SalesforceServiceCloud
+    dataTypes:
+      - SalesforceServiceCloud
+queryFrequency: 20m
```

Comment:
20m, seems closer to an NRT analytic.  If that is the case, then we should change it to that type.  If not, then have it query over a longer period of time and simply bin(TimeGenerated, 20m) the results so your failureCountThreshold still applies.

https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/binfunction